changelog_unreleased/api/10122.md

```diff
@@ -0,0 +1,34 @@
+#### Fix `lineSuffixBoundary` (#10122 by @thorn0)
+
+There was a bug in the implementation of the [`lineSuffixBoundary`][lsb] command that significantly limited its usefulness: the printer algorithm didn't correctly consider it a potential line break. Now that the bug has been fixed, we urge plugin authors to give this command another try and see if it can help them simplify printing of trailing comments.
```

Comment:
Why not `` [`lineSuffixBoundary`][https://github.com/prettier/prettier/blob/main/commands.md#linesuffixboundary] ``

Reply:
Just for readability/editability of non-rendered Markdown